Release Notes/Release Notes v2.11.0-beta1.md

```diff
@@ -0,0 +1,42 @@
+# .NET Driver Version 2.11.0-beta1 Release Notes
+
+This is a beta release for the 2.11.0 version of the driver.
+
+The main new features in 2.11.0-beta1 are:
+
+* .NET Standard 2.0 support
+* Support for `$out` to S3
+* Support for Snappy compression on .NET Core on Windows
+* Certificate revocation checking enabled by default and a new URI
+  option `tlsDisableCertificateRevocationCheck` to disable certificate
+  revocation checking
+
+An online version of these release notes is available at:
+
+https://github.com/mongodb/mongo-csharp-driver/blob/master/Release%20Notes/Release%20Notes%20v2.11.0-beta1.md
+
+The full list of JIRA issues that are currently scheduled to be resolved in this release is available at:
+
+https://jira.mongodb.org/issues/?jql=project%20%3D%20CSHARP%20AND%20fixVersion%20%3D%202.11.0%20ORDER%20BY%20key%20ASC
```

Comment:
I think it's correct, but just to highlight if I'm wrong. The provided jira link contains tickets which are not included into the current release

Reply:
I think this is correct: the link has traditionally shown issues scheduled to be in the final release, , as opposed to the issues that are in the beta release. :)